lib/folio_client/unexpected_response.rb

```diff
@@ -12,10 +12,12 @@ def self.call(response)
         raise ForbiddenError, "The operation requires privileges which the client does not have: #{response.body}"
       when 404
         raise ResourceNotFound, "Endpoint not found or resource does not exist: #{response.body}"
+      when 409
+        raise ConflictError, "Resource cannot be updated: #{response.body}"
```

Comment:
This looks uncovered which is making CodeClimate cranky. If you're open to it, you could prolly toss a quick 409 test in the authenticator spec which is currently covering the various exception classes. (Not the best place, maybe, but it's what we have now.)

Reply:
I think we could refactor that at some point, for sure.

Reply:
Added a spec.